build.sbt

```diff
@@ -6,7 +6,7 @@ organization := "com.kinja"
 
 version := "0.3.2" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
```

Comment:
I wasn't sure if we wanted to bump the version number since no actual functionality changed but I'll do that.